frontend/src/bundles/common/components/logo/logo.tsx

```diff
@@ -32,6 +32,32 @@ const Logo = ({
                     <img src={logoLabelSvg} alt="Label-BSATalents" />
                 </Grid>
             )}
+        </>
+    );
+};
+
+const Logo: React.FC<Properties> = ({
+    className = '',
+    isCollapsed = false,
+    link = { AppRoute },
+}) => {
+    return (
+        <Grid container>
+            {link === AppRoute.ROOT ? (
```

Comment:
this check does not work. it works only for the root page. I think it is better to pass some flag `withLink`.